README.md

```diff
@@ -44,16 +40,35 @@ every time, add the following alias to your `~/.bashrc`:
 alias fluo='fluo-dev fluo'
 ```
 
-Run the command below to copy Accumulo, Hadoop, and Zookeeper configuration in fluo-dev/conf
-to their respective installs.  This command will create a `fluo.properties` file at 
-fluo-dev/conf/fluo.properties that will be used by fluo when its deployed.
+Running Fluo using fluo-dev
+---------------------------
 
+With fluo-dev set up, you can now use it to download, configure, and run Fluo and its dependencies.
+
+First, run the command below to download the binary tarballs of Fluo dependencies (i.e Accumulo, Hadoop, 
+and Zookeeper). The command below downloads all tarballs and their corresponding file hashes and 
+signatures to the directory specified by `DOWNLOADS` in your env.sh. It will use the Apache download 
+mirror specified by `APACHE_MIRROR` in env.sh.  Other mirrors can be chosen from [this website][1].
+This command will also help you verify checksums and signatures (if you have `gpg` installed).
```

Comment:
Could point out that its important to inspect output related to hashes before proceeding.